graphql/resolvers.go

```diff
@@ -2594,6 +2594,12 @@ func (r *taskResolver) Status(ctx context.Context, obj *restModel.APITask) (stri
 	return *obj.DisplayStatus, nil
 }
 
+func (r *taskResolver) OriginalStatus(ctx context.Context, obj *restModel.APITask) (string, error) {
+	// this is needed for the task (as opposed to tasks) so that we don't overwrite
+	// the Status field in the cache, since tasks handles the status value differently
+	return *obj.DisplayStatus, nil
+}
```

Comment:
This does the same thing as the status resolver

Reply:
yes. We need the same thing without overwriting the cache.